catalog/core/catalog-core-solrcommands/src/main/java/org/codice/ddf/commands/solr/BackupCommand.java

```diff
@@ -133,7 +139,12 @@ public Object execute() throws Exception {
   private void performSingleNodeSolrBackup() throws URISyntaxException {
     verifySingleNodeBackupInput();
 
-    String backupUrl = getBackupUrl(coreName);
+    httpBuilder = new org.codice.solr.factory.impl.HttpClientBuilder(encryptionService);
```

Comment:
:question: any reason we're using the full package name here?

Reply:
Turns out that Apache also has a class named HttpClientBuilder. I didn't want anything getting confused about _which_ HttClientBuilder this was because they do not have the same interface.